app/assets/javascripts/clamp.js

```diff
@@ -0,0 +1,11 @@
+$(document).on('click','.read-more', function(e) {
+  e.stopPropagation();
+  e.preventDefault();
+  $(this).parent().find('.clamp').toggleClass('is-active');
+  if (this.innerHTML == 'See More...'){
+  	this.innerHTML = 'See Less...';
```

Comment:
I think it would be better to use jQuery accessors consistently - `$(this).html('See Less...')` etc. Maybe put `$(this)` in a variable, to save multiple constructor calls.